src/Navigation/Drawer/Drawer.test.tsx

```diff
@@ -28,6 +28,50 @@ describe('<Drawer /> render', () => {
       .toJSON();
     expect(snapshot).toMatchSnapshot();
   });
+
+  test('should match snapshot when position is right and drawer is open', () => {
+    const snapshot = renderer
+      .create(
+        <Drawer isOpen={true} position="right" onClose={jest.fn()}>
+          test-content
+        </Drawer>
+      )
+      .toJSON();
+    expect(snapshot).toMatchSnapshot();
+  });
+
+  test('should match snapshot when position is right and drawer is closed', () => {
+    const snapshot = renderer
+      .create(
+        <Drawer isOpen={false} position="right" onClose={jest.fn()}>
+          test-content
+        </Drawer>
+      )
+      .toJSON();
+    expect(snapshot).toMatchSnapshot();
+  });
+
+  test('should match snapshot when position is left and drawer is open', () => {
+    const snapshot = renderer
+      .create(
+        <Drawer isOpen={true} position="left" onClose={jest.fn()}>
+          test-content
+        </Drawer>
+      )
+      .toJSON();
+    expect(snapshot).toMatchSnapshot();
+  });
+
+  test('should match snapshot when position is left and drawer is closed', () => {
+    const snapshot = renderer
+      .create(
+        <Drawer isOpen={false} position="left" onClose={jest.fn()}>
+          test-content
+        </Drawer>
+      )
+      .toJSON();
+    expect(snapshot).toMatchSnapshot();
+  });
```

Comment:
@westwood846 , for these tests can we have just two tests for when the drawer is open and when it is closed, with dynamic positions.
Examplae:
```
describe('<Drawer /> prop position when drawer is open', () => {
  const matchSnapshot = (position: string) => {
    it(`should match snapshot when position = ${position} is passed`, () => {
      const { asFragment } = render(render the component);
      expect(asFragment()).toMatchSnapshot();
    });
  };

  ['left', 'right', undefined].forEach(matchSnapshot);
});
```

Reply:
Done!